Resources/Prototypes/_EinsteinEngines/Datasets/revolutionary_converter_speech.yml

```
@@ -0,0 +1,5 @@
+- type: localizedDataset
+  id: revolutionary_converter_speech
```

Different naming convention for IDs
```suggestion
  id: RevolutionaryConverterSpeech
```